openshift/system/Dockerfile

```diff
@@ -10,10 +10,8 @@ ADD . ./
 RUN cp config/examples/*.yml config/ \
     && cp openshift/system/config/* config/
 
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
-    && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* \
-    && dnf -y --setopt=module_stream_switch=True module enable ruby:${RUBY_VERSION} nodejs:18 mysql:8.0 \
-    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info make automake gcc gcc-c++ redhat-rpm-config postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel libxml2-devel libxslt-devel git 'dnf-command(download)' podman-catatonit \
+RUN dnf -y --setopt=module_stream_switch=True module enable ruby:${RUBY_VERSION} nodejs:18 mysql:8.0 \
+    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs --enablerepo=crb shared-mime-info make automake gcc gcc-c++ postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel libxml2-devel libxslt-devel git 'dnf-command(download)' podman-catatonit \
```

Comment:
There is a comment later in the file

> RHEL 9 simply use package `catatonit`

So we don't need `podman-catatonit` here. Should be removed. And then in `base` layer add `catatonit to the `dnf install` command. Also remove `COPY --from=builder /usr/libexec/catatonit/catatonit /usr/libexec/catatonit/catatonit`

Then maybe `ENTRYPOINT` will also need to be adjusted :angel:

Reply:
all else looks great!

Reply:
@akostadinov the entrypoint remained the same, the package installs the `catatonit` executable to `/usr/libexec/catatonit/catatonit`

Reply:
I hope it didn't pull the whole podman group as well :)